modules/clanmgr/Functions/CheckClanPW.php

```diff
@@ -8,8 +10,8 @@ function CheckClanPW($clanpw)
 {
     global $database;
 
-    $clan = $database->queryWithOnlyFirstRow("SELECT password FROM %prefix%clan WHERE clanid = ?", [$_GET['clanid']]);
-    if ($clan['password'] and $clan['password'] == md5($clanpw)) {
+    $clan = $db->qry_first("SELECT password FROM %prefix%clan WHERE clanid = %int%", $_GET['clanid']);
```

Comment:
Why is this query changed?
This can't work, as the global variable `$db` is not present.

My suggestion: Using the old database query code here
```suggestion
    $clan = $database->queryWithOnlyFirstRow("SELECT password FROM %prefix%clan WHERE clanid = ?", [$_GET['clanid']]);
```